web/modules/custom/pineapple_general/pineapple_general.module

```diff
@@ -97,6 +97,12 @@ function pineapple_general_form_alter(&$form, \Drupal\Core\Form\FormStateInterfa
   }
 }
 
+function pineapple_general_preprocess_webform(&$vars) {
```

Comment:
Update this to be variables instead of vars. Your site needs to be consistent. Also add a doc block comment above the hook

Reply:
done